design-system/materials/internals/definition.yaml

```diff
@@ -112,6 +112,11 @@ choiceGroupsByTheme:
         color-solid-10: 'hsl(0deg 0% 10% / 10%)'
         color-surface: '#fff'
         color-transparent: 'transparent'
+        color-success: 'hsl(152, 77%, 39%)'
+        color-success-25: 'hsl(155, 84%, 20%)'
+        color-success-40: 'hsl(155, 90%, 24%)'
+        color-success-85: 'hsl(144, 69%, 80%)'
+        color-success-95: 'hsl(141, 76%, 92%)'
```

Comment:
I've added the new `color-success` to the default theme as well. Otherwise we wouldn't be able to use `designTokens.colorSuccess85` since it won't be exported in the `design-tokens.d.ts` and we'd need to go for
```ts
import {
  themes,
} from '@commercetools-uikit/design-system';

themes['recolouring'].colorSuccess85,
```
instead